Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1763.cs

```diff
@@ -6,21 +6,40 @@
 using System.Runtime.CompilerServices;
 using System.Text;
 using System.Threading.Tasks;
-
 using Xamarin.Forms.CustomAttributes;
 using Xamarin.Forms.Internals;
 
+#if UITEST
+using NUnit.Framework;
+#endif
+
 namespace Xamarin.Forms.Controls.Issues
 {
 	[Preserve (AllMembers=true)]
 	[Issue (IssueTracker.Github, 1763, "First item of grouped ListView not firing .ItemTapped", PlatformAffected.WinPhone, NavigationBehavior.PushAsync)]
-	public class Issue1763 : TabbedPage
+	public class Issue1763 : TestTabbedPage
 	{
 		public Issue1763 ()
+		{
+		
+		}
+
+		protected override void Init()
 		{
 			Title = "Contacts";
+			Children.Add(new ContactsPage());
+		}
 
-			Children.Add (new ContactsPage ());
+#if UITEST && __WINDOWS__
+		[Test]
+		public void TestIssue1763ItemTappedFiring()
+		{
+			RunningApp.WaitForElement(q => q.Marked("Contacts"));
+			RunningApp.Tap(q => q.Marked("Egor1"));
+			RunningApp.WaitForElement(q => q.Marked("Tapped a List item"));
+			RunningApp.Tap(q => q.Marked("Destruction"));
```

Comment:
Can we wait for something to happen after destruction to ensure it did whatever it's supposed to do?

Reply:
Touch

Reply:
changed by using WaitForNotElement